corehq/apps/case_search/templates/case_search/bootstrap5/case_search.html

```diff
@@ -20,29 +20,37 @@ <h3>{% trans "Search Parameters" %}</h3>
     <div class="row">
       <div class="col-md-12">
         <div class="controls">
-          <form class="form-inline" id="case-search-form">
+          <form id="case-search-form">
             {% csrf_token %}
-            <div class="card card-default">
+            <div class="card card-default mb-3">
               <div class="card-header">
                 <strong>{% trans "Case Metadata" %}</strong>
               </div>
-              <div class="card-body">
-                <label>
-                  {% trans "Case Type:" %}
+              <div class="card-body row">
+                <div class="col-auto">
```

Comment:
I am curious if the nesting here is necessary. When I look at the [documentation](https://getbootstrap.com/docs/5.0/forms/layout/#horizontal-form-label-sizing) is looks like `col-auto` could just go on the label and you could remove this extra div level